packages/hiro-graph-client/src/eventstream.js

```diff
@@ -32,7 +32,7 @@ export default class EventStream {
         this._token = token;
         this._endpoint = endpoint
             .replace(/^http/, "ws") //replace http(s) with ws(s)
-            .replace(/\/?$/, "/api/6.1/events-ws/?"); // replace possible trailing slash with api endpoint
```

Comment:
does this work? Before it was just `_events`

Reply:
Ah right, I reverted your changes. But I think you did earlier ones too.